PyInstaller/building/api.py

```diff
@@ -780,10 +780,11 @@ def _merge_dependencies(self, args):
         Filter shared dependencies to be only in first executable.
         """
         for analysis, _, _ in args:
-            path = os.path.abspath(analysis.scripts[-1][1]).replace(self._common_prefix, "", 1)
+            path = os.path.normcase(os.path.abspath(analysis.scripts[-1][1]))
+            path = path.replace(self._common_prefix, "", 1)
             path = os.path.splitext(path)[0]
-            if os.path.normcase(path) in self._id_to_path:
-                path = self._id_to_path[os.path.normcase(path)]
+            if path in self._id_to_path:
+                path = self._id_to_path[path]
```

Comment:
Is this actually needed? Please state in the commit message, why. If it is just for code clean up, I'd omit this change,

Reply:
Not needed, was just removing the redundancy of the extra normcase. I have removed this change